### ISSUE TITLE
Token running out

### ISSUE THREAD

**Issue by Setup007** [nmynarcik/youtube-sub-alert#6]
Hello there!
I do not have a fix for the token running out yet, since this is a client side application.
Any ideas how to port everything to a server side application so that we can store the token and refresh it when needed?
Imagine having the authorization window popping out when you are streaming a game... That would be very annoying.

**Comment by nmynarcik** [comment]
Yeah, my test streamer had this issue. Could you leverage express and have it do this logic for us on the backend?

**Comment by Setup007** [comment]
To be honest with you, I discovered npm just yesterday. So I really do not know yet what would needed to be done to support this :/
Plus as of the 1st of September 2016 [this website here](https://www.livestreamalerts.com/youtubesubs/) apparently supports all what you were trying to achieve...

**Comment by nmynarcik** [comment]
HAHAHA wow! Awesome...thanks for sharing!
